GameData/RealismOverhaul/Engine_Configs/BE-4_Config.cfg

```diff
@@ -37,7 +38,7 @@
 //	* No TestFlight compatibility since this global engine config represents unflown hardware.
 //	==================================================
 
-@PART[*]:HAS[#engineType[BE4]]:FOR[RealismOverhaulEngines]
+@PART[*]:HAS[#engineType[BE-4]]:FOR[RealismOverhaulEngines]
```

Comment:
If you're going to rename the engine type with a -, you must find any other engines that used the old reference and update them.
Hint: RO_VNP_Engines.cfg has one.  So is a KK launcher.